cmd/cloud.go

```diff
@@ -343,20 +344,69 @@ func getCmdCloud(gs *state.GlobalState) *cobra.Command {
 	}
 
 	exampleText := getExampleText(gs, `
-  {{.}} cloud script.js`[1:])
+  # Authenticate with Grafana k6 Cloud
+  $ {{.}} cloud login
+
+  # Run a k6 script in the Grafana k6 cloud
+  $ {{.}} cloud run script.js
+
+  # Run a k6 archive in the Grafana k6 cloud
+  $ {{.}} cloud run archive.tar
+
+  # [deprecated] Run a k6 script in the Grafana k6 cloud
+  $ {{.}} cloud script.js
+
+  # [deprecated] Run a k6 archive in the Grafana k6 cloud
+  $ {{.}} cloud archive.tar`[1:])
 
 	cloudCmd := &cobra.Command{
 		Use:   "cloud",
 		Short: "Run a test on the cloud",
-		Long: `Run a test on the cloud.
+		Long: `[deprecation notice]
+The k6 team is in the process of modifying and deprecating the cloud command behavior. In the future, the "cloud"
+command will only display a help text, instead of running tests in the cloud.
+To run tests in the cloud, users are now invited to migrate to the "k6 cloud run" command instead.
+
+Run a test on the cloud.
 
 This will execute the test on the k6 cloud service. Use "k6 login cloud" to authenticate.`,
```

Comment:
```suggestion
This will execute the test on the k6 cloud service. Use "k6 cloud login" to authenticate.`,
```

Reply:
At this point, I think we can start suggesting the new commands in help messages, right?